tests/tt_metal/tt_metal/unit_tests_common/dprint/test_print_before_finish.cpp

```diff
@@ -58,11 +58,7 @@ static void RunTest(DPrintFixture* fixture, Device* device) {
 }
 
 TEST_F(DPrintFixture, TestPrintFinish) {
-    // Iterate devices in reverse for this test. Since the test closes the devices early, avoid
-    // closing L chip before R chip.
     auto devices = this->devices_;
-    std::reverse(devices.begin(), devices.end());
-    for (Device* device : devices) {
-        this->RunTestOnDevice(RunTest, device);
-    }
+    // Run only on the first device
```

Comment:
Can we just comment here that we run only on first device because the test closes devices as it goes, and that causes problems for multi-chip?